src/server_manager/web_app/shadowbox_server.ts

```diff
@@ -35,9 +34,27 @@ export interface ServerConfig {
   accessKeyDataLimit?: server.DataLimit;
```

Comment:
We should apply the same here. I scanned the file and this is the last place we rely on the model in Shadowbox API types.

Reply:
I'll do that in a separate PR